lib/caseStatus.spec.ts

```diff
@@ -49,6 +49,23 @@ describe('case status APIs', () => {
     });
   });
 
+  describe('updateCaseStatus', () => {
+    it("calls the service API's update POST case status endpoint", async () => {
+      mockedAxios.post.mockResolvedValue({ data: {} });
+      await caseStatusAPI.updateCaseStatus({
+        data: mockedCaseStatusAddRequest,
+      });
+      expect(mockedAxios.post).toHaveBeenCalled();
+      expect(mockedAxios.post.mock.calls[0][0]).toEqual(
+        `${ENDPOINT_API}/residents/case-statuses`
```

Comment:
This needs to follow the new endpoint defined in SwaggerHub 

/case-statuses/{id}/answers